client/components/password-change/password-change.js

```diff
@@ -0,0 +1,75 @@
+import "./index.css";
+import React, {useState} from "react";
+import axios from "axios";
+import qs from "qs";
+import PropTypes from "prop-types";
+
+import {passwordChangeApiUrl} from "../../constants";
+
+const PasswordChangeComponent = ({orgSlug}) => {
+  const [newPassword1, changeNewPassword1] = useState();
+  const [newPassword2, changeNewPassword2] = useState();
+
+  const handleSubmit = () => {
+    const url = passwordChangeApiUrl.replace("{orgSlug}", orgSlug);
+    return axios({
+      method: "post",
+      headers: {
+        "content-type": "application/x-www-form-urlencoded",
+      },
+      url,
+      data: qs.stringify({
+        newPassword1,
+        newPassword2,
+      }),
+    }).catch(error => {
+      console.log(error);
+    });
+  };
+
+  return (
+    <div className="owisp-passchange-container">
+      <div className="title">Change your password</div>
```

Comment:
@someshkar  Please do not hard code anything. Everything should be defined in the [default-config.yml](https://github.com/openwisp/openwisp-wifi-login-pages/blob/master/org-configurations/default-configuration.yml) and you should render fields, inputs, titles, placeholders, buttons etc accordingly. `Change your password` should come from the config file and should change to different language when the language is switched.